code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -86,7 +86,9 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/giant_spider/mob_negates_gravity()
-	return (locate(/obj/structure/spider/stickyweb) in loc) || ..()
```

Comment:
wait what how does this not return a bool? 
"Hey is value x or y true?" 
"Spiderweb"

Reply:
...somehow. My guess would be that if ..() is false then || just returns the other one?

Reply:
wack

Reply:
`x || y` resolves to x if its truthy, y if it's not

Reply:
Bools in BYOND don't exist, this was returning a truthy value

Reply:
This is a really useful behavior to know, and we use it a lot for things like default values, such as `var/name = input_name || "Other value"`